i18n/hu.yaml

```diff
@@ -0,0 +1,90 @@
+# General
+- id: read_more
+  translation: Tovább olvas…
+
+- id: menu_label
+  translation: Menu
+
+- id: sidemenu_title
+  translation: Menu
+
+# Post meta
+- id: meta_lastmod
+  translation: Utoljára módosítva
+
+- id: meta_translations
+  translation: Fordítások
+
+# Table of Contents
+- id: toc_title
+  translation: Az oldal tartalma
+
+# Post nav
+- id: post_nav_prev
+  translation: Előző
+
+- id: post_nav_next
+  translation: Következő
+
+# Authorbox
+- id: authorbox_name
+  translation: A {{ .Count }} -ról
+
+# Sidebar
+- id: sidebar_warning
+  translation: FIGYELMEZTETÉS
+
+- id: sidebar_recommendation
+  translation: Kérjük, aktiváljon legalább egy oldalsáv-modult.
+
+# Search widget
+- id: search_placeholder
+  translation: KERES...
+
+# Languages widget
+- id: languages_title
+  translation: Nyelvek
+
+# Categories widget
+- id: categories_title
+  translation: Kategóriák
+
+# Recent Posts widget
+- id: recent_title
+  translation: Legutóbbi bejegyzések
+
+# Social widget
+- id: social_title
+  translation: Közösség
+
+# Tags List widget
+- id: tags_title
+  translation: Cimkék
+
+# Footer
+- id: footer_credits
+  translation: "Létrehozva <a href=\"https://gohugo.io/\" rel=\"nofollow noopener\" target=\"_blank\">Hugo</a> és \
+  <a href=\"https://github.com/Vimux/Mainroad/\" rel=\"nofollow noopener\" target=\"_blank\">Mainroad</a> témával."
+
+# 404
+- id: 404_title
+  translation: 404. Az oldal nem található
+
+- id: 404_text
+  translation: "Úgy tűnik, hogy a kívánt oldalt áthelyezték, törölték vagy nem létezik. Kérjük, használja a keresést, vagy lépjen ide"
+
+- id: 404_linktext
+  translation: "főoldal"
+
+# No posts empty state
+- id: noposts_warning_title
+  translation: "Még nincs bejegyzésed!"
+
+- id: noposts_warning_description
+  translation: Ha valamit a <b>content</b> könyvtár alatti mappába (szakaszba) közzéteszel, az itt jelenik meg. Csak egy szakasz \
```

Comment:
```
Error: add site dependencies: load resources: loading translations: "/Users/vimux/code/mainroad.dev/themes/mainroad/i18n/hu.yaml:87:1": failed to load translations: failed to unmarshal hu.yaml: yaml: line 87: could not find expected ':'
```

Translation string [L84-85](https://github.com/Vimux/Mainroad/blob/cf3666d8bcc36825bcecdabd2007b2e8c5373352/i18n/hu.yaml#L84-L85) needs quotes.

Reply:
Dear Vimux,

You were right, it's been fixed now!

Cheers!